### PR TITLE
post GA deprecate 3.9 builds

### PR DESCRIPTION
**Pull request by mjudeikis**
Deprecate 3.9 builds jobs post 3.10 GA

cc: @openshift/sig-azure

### CODE REVIEW

**Comment by kargakis** [comment]
It sees I have lost my ability to merge in this repo :/

**Comment by mjudeikis** [comment]
@stevekuznetsov ping. Merge please :) or make @kargakis  all powerfull again please :)

**Comment by kwoodson** [comment]
@mjudeikis, please rebase/update and let's get this merged.  No reason for 3.9

File: sjb/config/test_cases/azure_build_node_image_rhel.yml [hunk 1]
```diff
@@ -41,13 +41,12 @@ actions:
     parameters:
       - JOB_SPEC
       - AZURE_VM_SIZE
-      - OPENSHIFT_RELEASE
 
   - type: "host_script"
     title: "determine the resource group"
     script: |-
       echo "RESOURCE_GROUP=ci-$JOB_NAME-$BUILD_NUMBER" | ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel "cat >>/etc/environment"
-      echo "IMAGE_NAME=rhel7-$OPENSHIFT_RELEASE-$(TZ=Etc/UTC date +%Y%m%d%H%M)" | ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel "cat >>/etc/environment"
```
Comment (jim-minter) [inline sjb/config/test_cases/azure_build_node_image_rhel.yml]: revert this line

File: sjb/config/test_cases/azure_build_node_image_rhel.yml [hunk 2]
```diff
@@ -41,13 +41,12 @@ actions:
     parameters:
       - JOB_SPEC
       - AZURE_VM_SIZE
-      - OPENSHIFT_RELEASE
```
Comment (jim-minter) [inline sjb/config/test_cases/azure_build_node_image_rhel.yml]: this line may be needed too

**Pull request merged**